src/Compilers/CSharp/Portable/Symbols/ConstantValueUtils.cs

```diff
@@ -155,5 +157,21 @@ internal static string UnescapeInterpolatedStringLiteral(string s)
 
             return value;
         }
+
+        private sealed class CheckConstantInterpolatedStringValidity : BoundTreeWalkerWithStackGuardWithoutRecursionOnTheLeftOfBinaryOperator
+        {
+            internal DiagnosticBag diagnostics;
+
+            public CheckConstantInterpolatedStringValidity(DiagnosticBag diagnostics)
+            {
+                this.diagnostics = diagnostics;
+            }
+
+            public override BoundNode VisitInterpolatedString(BoundInterpolatedString node)
+            {
+                Binder.CheckFeatureAvailability(node.Syntax, MessageID.IDS_FeatureConstantInterpolatedStrings, diagnostics);
+                return base.VisitInterpolatedString(node);
+            }
```

Comment:
We don't need to continue recursion. Just return `null` and it's good.

Reply:
Thanks for the reminder! Fixed